app/editor/src/store/hooks/admin/products/useProducts.ts

```diff
@@ -1,7 +1,7 @@
 import { IProductModel, useApiAdminProducts } from 'hooks/api-editor';
 import React from 'react';
 import { useApiDispatcher } from 'store/hooks';
-import { IAdminState, useAdminStore } from 'store/slices';
+import { ILookupState, useLookupStore } from 'store/slices';
```

Comment:
Incorrect.  The admin section is separate by design.  You will however need to update lookups appropriately when changes occur.  Note that there are two different types of challenges here.
a) Admin edits values and their current session should update their local lookups
b) Other users have no idea that things have changed unless they do a hard refresh, as their lookups are cached for a time.

Reply:
Sounds good - thanks for the help!

Reply:
You shouldn't have to worry about issue (b).  If a user has some stale content they'll have to live with that until their cache expires.

Reply:
@Fosol updated and also added TNO-457 :)